apps/BosGenaDrop/widget/Mintbase/App/Activity/Activity.jsx

```diff
@@ -0,0 +1,278 @@
+      color: ${isDarkModeOn ? "#c2cdfd" : "#4e58a2"};
+      svg {
+        box-sizing: content-box;
+        height: 14px;
+        color: ${isDarkModeOn ? "#c2cdfd" : "#4e58a2"};
+        cursor: pointer;
+        padding: 10px;
+        border-radius: 2px;
+        transition: all 200ms ease 0s;
+        :hover {
+          fill: white;
+          background: '';
+        }
+      }
+    }
+  }
+
+  .price {
+    display: flex;
+    gap: 4px;
+    align-items: center;
+    font-weight: 600;
+    color: ${isDarkModeOn ? "#c2cdfd" : "#4e58a2"};
+    img {
+      width: 14px;
+    }
+  }
+
+  @media (max-width: 500px) {
+    .header,
+    .trx-row {
+      grid-template-columns: repeat(6, 150px);
+    }
+  }
+`;
+
+const Row = styled.div`
+    margin-top:20px;
+    display:flex;
+    flex-direction: row; 
+    gap:20px;
+    flex-wrap: nowrap;
+    width:100%;
+    justify-content: center;
+    padding: 10px 20px;
+`;
+
+const ContainerCard = styled.div`
+  .info-card {
+    border-radius: 0.25rem;
+    background: ${IsDarkModeOn ? "var(--gray-800)" : "rgba(40,42,58,0.1)"};
+    &.small,
+    &.medium,
+    &.big {
+      padding: 8px;
+      min-height: 46px;
+      @media (min-width: 480px) {
+        min-height: 72px;
+        padding: 12px;
+      }
+    }
+    @media (min-width: 976px) {
+      .info-card.small {
+        min-width: 160px;
+      }
+    }
+    .title-wrapper {
+      display: flex;
+      position: relative;
+      align-items: center;
+    }
+    
+
+`;
+
+const Title = styled.div`
+  color: ${IsDarkModeOn ? "var(--gray-300)" : "var(--gray-700)"};
+  font-size: 13px;
+`;
+const Description = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 100%;
+  magrin: 5px 0;
+  .description-rt {
+    display: flex;
+    width: 100%;
+    font-size:18px;
+    font-weight: 600;
+  }
+`;
+
+
+const MbCard = ({title,description})=>{
+    return(
+        title&&description&&
+        <ContainerCard>
+            <div className="info-card medium">
+                <div className={`title-wrapper`}>
+                    <Title>{title}</Title>
+                </div>
+                <Description>
+                    <div className="description-rt">{description}</div>
+                </Description>
+            </div>     
+        </ContainerCard>
+    )
+}
+
+const data = [
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Transactions (All Near)",
+        description: "4,644,3773"
+    },
```

Comment:
All these are dummy data, could you reach out to mintbase team and request the API being called on that page. Let me know if you need help reaching out

Reply:
I can't reach out to mintbase team, Can you reach out to mintbase and Can help me request the API?